lib/features/authentication/core/data/datasource/authentication_remote_data_source.dart

```diff
@@ -0,0 +1,30 @@
+part of '_datasource.dart';
+
+abstract class AuthenticationRemoteDataSource {
+  Future<User> signIn();
+}
+
+@Injectable(as: AuthenticationRemoteDataSource)
+class AudiometerRemoteDataSourceImpl implements AuthenticationRemoteDataSource {
```

Comment:
sorry, what is `Audiometer` in this context? I'm confused because I can't seem to find any reference to this class

Reply:
Sorry, it's a mistake, going to fix it ASAP.

Reply:
FIXED